packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -48,14 +48,73 @@ const DocumentStatus = ({ status = 'draft' }: DocumentStatusProps) => {
   );
 };
 
+const WidgetContent = ({ document }: { document: RecentDocument }) => {
+  const { formatMessage } = useIntl();
+  const { trackUsage } = useTracking();
+  const navigate = useNavigate();
+
+  const handleRowClick = (document: RecentDocument) => () => {
+    trackUsage('willEditEntryFromHome');
+    const link = getEditViewLink(document);
+    navigate(link);
+  };
+
+  return (
+    <Tr onClick={handleRowClick(document)} key={document.documentId}>
+      <Td>
+        <CellTypography title={document.documentId} variant="omega" textColor="neutral800">
+          {document.title}
+        </CellTypography>
+      </Td>
+      <Td>
+        <CellTypography variant="omega" textColor="neutral600">
+          {document.kind === 'singleType'
+            ? formatMessage({
+                id: 'content-manager.widget.last-edited.single-type',
+                defaultMessage: 'Single-Type',
+              })
+            : formatMessage({
+                id: document.contentTypeDisplayName,
+                defaultMessage: document.contentTypeDisplayName,
+              })}
```

Comment:
@remidej this is how we are doing it  in the left menu https://github.com/strapi/strapi/blob/e843ecdd507f6e96bfcd9dc614af6cd103720413/packages/core/content-manager/admin/src/components/LeftMenu.tsx#L93-L96

It means the translation is at the root of the config

```
const config = {
  locales: ['it', 'es', 'en', 'en-GB'],
  translations: {
    en: {
      'Kitchen Sink': 'Baguette',    
    },
  },
};
```